### PR TITLE
Enhance mempool to consider gas fees to miner

### PR DESCRIPTION
**Pull request by lucafavatella**
Delivers https://www.pivotaltracker.com/story/show/154664693

Follows from #1232 

TODO:
* [x] Check minimum gas price before adding tx to mempool
* [x] Enhance mempool to consider reward miner might get by processing contract-related transaction
  * [x] Tests
  * [x] Code
* [x] Ensure Python UAT tests pass
  * [CI job passed](https://circleci.com/gh/lucafavatella/epoch/4178) on f2a46806 equivalent to 7e6a954f
* [x] Ensure system tests pass
  *  [CI job passed](https://circleci.com/gh/lucafavatella/epoch/4174) on f2a46806 equivalent to 7e6a954f 
  * (System tests had [passed](https://circleci.com/gh/aeternity/epoch/16911) on master 5a138412 )

### CODE REVIEW

File: apps/aecore/src/aec_tx_pool.erl [hunk 1]
```diff
@@ -357,3 +371,21 @@ check_minimum_fee(Tx,_Hash) ->
...
+int_check_minimum_gas_price(GasPrice) ->
+    case GasPrice >= aec_governance:minimum_gas_price() of
+        true  -> ok;
+        false -> {error, too_low_gas_price}
+    end.
+
+int_gas_price(Tx) ->
+    case aetx:lookup_gas_price(Tx) of
+        none -> ?PSEUDO_GAS_PRICE;
+        {value, GP} when is_integer(GP), GP >= 0 -> GP
```
Comment (hanssv) [inline apps/aecore/src/aec_tx_pool.erl]: This should only be called for proper Txs so this check is overly cautious?

File: deployment/ansible/deploy.yml [hunk 2]
```diff
@@ -56,7 +56,7 @@
           port: 3114
       chain:
         persist: true
-        db_path: "./db64"
+        db_path: "./db65"
```
Comment (hanssv) [inline deployment/ansible/deploy.yml]: Is the mempool sorting part of consensus? But better bump one time too many!

Comment (lucafavatella) [reply deployment/ansible/deploy.yml]: Mmm... I was thinking the mempool key. Though that table is not persisted so I could have avoided.

One more no issue though.

File: apps/aecore/src/aec_tx_pool.erl (hunk 1 above)
Comment (lucafavatella) [reply apps/aecore/src/aec_tx_pool.erl]: Correct.

**Pull request merged**